Fix MSVC linker error LNK2005 due to multiple definitions

```
error LNK2005: "void * linenoise::ansi::hConOut" (?hConOut@ansi@linenoise@@3PEAXEA) already defined in main.obj 
error LNK2005: "int linenoise::ansi::state" (?state@ansi@linenoise@@3HA) already defined in main.obj    
error LNK2005: "wchar_t linenoise::ansi::prefix" (?prefix@ansi@linenoise@@3_WA) already defined in main.obj  
error LNK2005: "wchar_t linenoise::ansi::prefix2" (?prefix2@ansi@linenoise@@3_WA) already defined in main.obj 
error LNK2005: "wchar_t linenoise::ansi::suffix" (?suffix@ansi@linenoise@@3_WA) already defined in main.obj 
error LNK2005: "int linenoise::ansi::es_argc" (?es_argc@ansi@linenoise@@3HA) already defined in main.obj 
error LNK2005: "int * linenoise::ansi::es_argv" (?es_argv@ansi@linenoise@@3PAHA) already defined in main.obj 
error LNK2005: "wchar_t * linenoise::ansi::Pt_arg" (?Pt_arg@ansi@linenoise@@3PA_WA) already defined in main.obj 
error LNK2005: "int linenoise::ansi::Pt_len" (?Pt_len@ansi@linenoise@@3HA) already defined in main.obj  
error LNK2005: "int linenoise::ansi::shifted" (?shifted@ansi@linenoise@@3HA) already defined in main.obj 
error LNK2005: "struct linenoise::ansi::GRM linenoise::ansi::grm" (?grm@ansi@linenoise@@3UGRM@12@A) already defined in main.obj 
error LNK2005: "struct _COORD linenoise::ansi::SavePos" (?SavePos@ansi@linenoise@@3U_COORD@@A) already defined in main.obj 
error LNK2005: "int linenoise::ansi::nCharInBuffer" (?nCharInBuffer@ansi@linenoise@@3HA) already defined in main.obj 
error LNK2005: "wchar_t * linenoise::ansi::ChBuffer" (?ChBuffer@ansi@linenoise@@3PA_WA) already defined in main.obj 
error LNK2005: "void * linenoise::hOut" (?hOut@linenoise@@3PEAXEA) already defined in main.obj 
error LNK2005: "void * linenoise::hIn" (?hIn@linenoise@@3PEAXEA) already defined in main.obj 
error LNK2005: "unsigned long linenoise::consolemodeIn" (?consolemodeIn@linenoise@@3KA) already defined in main.obj 
```

--- a/linenoise.hpp
+++ b/linenoise.hpp
@@ -187,7 +187,7 @@ namespace ansi
 
 	// ========== Global variables and constants
 
-	HANDLE hConOut; // handle to CONOUT$
+	inline HANDLE hConOut; // handle to CONOUT$
 
 	const char ESC = '\x1B'; // ESCape character
 	const char BEL = '\x07';
@@ -195,15 +195,15 @@ namespace ansi
 	const char SI = '\x0F'; // Shift In
 
 	const int MAX_ARG = 16; // max number of args in an escape sequence
-	int state; // automata state
-	WCHAR prefix; // escape sequence prefix ( '[', ']' or '(' );
-	WCHAR prefix2; // secondary prefix ( '?' or '>' );
-	WCHAR suffix; // escape sequence suffix
-	int es_argc; // escape sequence args count
-	int es_argv[MAX_ARG]; // escape sequence args
-	WCHAR Pt_arg[MAX_PATH * 2]; // text parameter for Operating System Command
-	int Pt_len;
-	BOOL shifted;
+	inline int state; // automata state
+	inline WCHAR prefix; // escape sequence prefix ( '[', ']' or '(' );
+	inline WCHAR prefix2; // secondary prefix ( '?' or '>' );
+	inline WCHAR suffix; // escape sequence suffix
+	inline int es_argc; // escape sequence args count
+	inline int es_argv[MAX_ARG]; // escape sequence args
+	inline WCHAR Pt_arg[MAX_PATH * 2]; // text parameter for Operating System Command
+	inline int Pt_len;
+	inline BOOL shifted;
 
 	// DEC Special Graphics Character Set from
 	// http://vt100.net/docs/vt220-rm/table2-4.html
@@ -289,17 +289,17 @@ namespace ansi
 				7 // white
 			};
 
-	GRM grm;
+	inline GRM grm;
 
 	// saved cursor position
-	COORD SavePos;
+	inline COORD SavePos;
 
 	// ========== Print Buffer functions
 
 #define BUFFER_SIZE 2048
 
-	int nCharInBuffer;
-	WCHAR ChBuffer[BUFFER_SIZE];
+	inline int nCharInBuffer;
+	inline WCHAR ChBuffer[BUFFER_SIZE];
 
 	//-----------------------------------------------------------------------------
 	//   FlushBuffer()
@@ -938,9 +938,9 @@ namespace ansi
 
 } // namespace ansi
 
-HANDLE hOut;
-HANDLE hIn;
-DWORD consolemodeIn = 0;
+inline HANDLE hOut;
+inline HANDLE hIn;
+inline DWORD consolemodeIn = 0;
 
 inline int win32read(int *c)
 {

